index.html

```diff
@@ -1,307 +1,278 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./style.css" />
-    <title>Aubin SIMPEZE - Full-Sack Web Developer</title>
-  </head>
 
-  <body>
-    <header class="header-container">
-      <a href="index.html" class="logo">AubinLeYoung</a>
-      <nav class="nav-container">
-        <button class="menu-btn" type="submit">
-          <img src="./assets/icons/menu.png" alt="" />
-        </button>
-        <ul class="portfolio-menu">
-          <li><a href="index.html">Portfolio</a></li>
-          <li><a href="#about-me-container">About</a></li>
-          <li><a href="#contact">Contact</a></li>
-          <li>
-            <a href="mailto:aubinsimpeze01@gmail.com"
-              ><img src="assets/icons/email_icone.png" alt=""
-            /></a>
-          </li>
-        </ul>
-      </nav>
-    </header>
-    <section class="headline-container">
-      <h1 class="white headline-text">
-        <span class="headline-title"
-          >Hey There <br />
-          I'm Aubin</span
-        ><br />
-        I am a Software Developer
-      </h1>
-      <p class="white headline-p">
-        I can help you build a product , feature or website Look through some of
-        my work andexperience! If you like what you see and have a project you
-        need coded, don’t hestiate to contact me.
-      </p>
-      <ul class="social-media-icones-container">
-        <li>
-          <a href="https://github.com/aubinleyoung"
-            ><img src="assets/icons/1.png" alt="Github link"
-          /></a>
-        </li>
-        <li>
-          <a href="https://www.linkedin.com/in/aubin-simpeze-7a5b7a220/"
-            ><img src="assets/icons/2.png" alt="LinkedIn link"
-          /></a>
-        </li>
-        <li>
-          <a href=""><img src="assets/icons/3.png" alt="Angelist link" /></a>
-        </li>
-        <li>
-          <a href="https://twitter.com/SimpezeAubin"
-            ><img src="assets/icons/4.png" alt="Twitter link"
-          /></a>
-        </li>
-        <li>
-          <a href=""><img src="assets/icons/5.png" alt="Medium-link" /></a>
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./style.css" />
+
+
```

Comment:
- [ ] It's great that you have a license file in your project but it looks like a template 🤔 Kindly add your **name**, **current year** and **app-type** 😊 to make your project licensed to you 👍

### After doing it your MIT.md file should have the below code 😊 ⬇️

```md
## Copyright 2022, Aubin SIMPEZE

Permission is hereby granted, free of charge, to any person obtaining a copy of this WEB APP and associated documentation files, to deal in the WEB APP without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the WEB APP, and to permit persons to whom the WEB APP is furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all copies or substantial portions of the WEB APP.

THE WEB APP IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE WEB APP OR THE USE OR OTHER DEALINGS IN THE WEB APP.
```